addon/helpers/bs-action-fallback.js

```diff
@@ -0,0 +1,9 @@
+import { helper } from '@ember/component/helper';
+
+const noop = () => {};
+
+export function bsActionFallback(params) {
+  return params[0] ?? noop;
+}
```

Comment:
This reminds me that I wanted to propose having `{{fn}}` helper to return a noop function if called without any argument.

Reply:
Yeah, something to solve this upstream really makes sense IMO. Or `{{on ...}}` ignoring an `undefined` callback or something like that. You hereby have my support for writing an RFC! 😝

Reply:
It's discussed here: https://github.com/emberjs/rfcs/issues/660

Reply:
I expect `{{or @actionHandler (fn)}}` to be the syntax supported by Ember out of the box in mid-term. Wondering if we should use a `{{bs-noop}}` together with `{{bs-default}}` instead of `{{bs-action-fallback}}` as an interim solution. Refactoring would be a task of search and replace in that case.

Reply:
Published a RFC, which proposes to support an argumentless version of `{{fn}}` helper to create a noop function as we had discussed here: https://github.com/emberjs/rfcs/pull/676

Reply:
> Wondering if we should use a {{bs-noop}} together with {{bs-default}} instead of {{bs-action-fallback}} as an interim solution

Makes sense, updated.